src/custom/state/orders/updaters/utils.ts

```diff
@@ -21,14 +21,18 @@ export function computeOrderSummary({
   // if we can find the order from the API
   // and our specific order exists in our state, let's use that
   if (orderFromApi) {
-    const { buyToken, sellToken, sellAmount, buyAmount, executedBuyAmount, executedSellAmount } = orderFromApi
+    const { buyToken, sellToken, sellAmount, feeAmount, buyAmount, executedBuyAmount, executedSellAmount } =
+      orderFromApi
 
     if (orderFromStore) {
       const { inputToken, outputToken, status, kind } = orderFromStore
       const isFulfilled = status === OrderStatus.FULFILLED
 
       // don't show amounts in atoms
-      const inputAmount = stringToCurrency(isFulfilled ? executedSellAmount : sellAmount, inputToken)
+      const inputAmount = isFulfilled
```

Comment:
i'm concerned WHY this is all changing. this was a very long back and forth that really was reverted in 1 change in 1 version? are we sure this is the route we want to go?

Reply:
do we have access to the `trade` object here? if so we could rework it to accept that and use the `inputAmountWithFree` (or whatever the name is)

Reply:
> i'm concerned WHY this is all changing. this was a very long back and forth that really was reverted in 1 change in 1 version? are we sure this is the route we want to go?

I suspect this has been like that since the beginning when I implemented this and we never really bothered to fix it

OR

The slippage changes changed what's stored in the `sellAmount`

But I really think it's the former.
I can look at how previous versions behave on this matter but don't think it's worth the effort

> do we have access to the trade object here?

We don't, as this is in the middleware, using a combination of data from redux and API